src/icesat2_tracks/analysis_db/B03_plot_spectra_ov.py

```diff
@@ -0,0 +1,581 @@
+import os, sys
+
+"""
+This file open a ICEsat2 track applied filters and corections and returns smoothed photon heights on a regular grid in an .nc file.
+This is python 3
+"""
```

Comment:
The comment should come right at the start of the file, so I'd switch these lines around:
```suggestion
"""
This file open a ICEsat2 track applied filters and corections and returns smoothed photon heights on a regular grid in an .nc file.
This is python 3
"""

import os, sys
```

Reply:
`os` wasnt used in the file